install.py

```diff
@@ -34,3 +34,13 @@ def _subprocess_call(cmd):  # this is the python 2.7 implementation
 )
 print()
 print("✔️ Detect-secrets hook installed")
+
+_subprocess_call(
+    [
+        "bash",
+        os.environ["HOME"] + "/.gds-pre-commit/register.sh",
```

Comment:
this no longer is how this file is used, @akinnane and @OllieJC made the repo relocatable